test/uk/gov/hmrc/filetransmission/utils/TestHttpClient.scala

```diff
@@ -18,17 +18,25 @@ package uk.gov.hmrc.filetransmission.utils
 
 import akka.actor.ActorSystem
 import akka.stream.ActorMaterializer
-import com.typesafe.config.Config
+import com.typesafe.config.{Config, ConfigFactory}
 import play.api.libs.ws.ahc.AhcWSClient
-import uk.gov.hmrc.play.bootstrap.http.HttpClient
+import uk.gov.hmrc.http.HttpClient
+import uk.gov.hmrc.http.hooks.HttpHook
 import uk.gov.hmrc.play.http.ws.WSHttp
+import scala.collection.JavaConverters._
 
 class TestHttpClient extends HttpClient with WSHttp {
```

Comment:
you can use `HttpClientSupport` trait from `http-verbs-test`
the configuration defaults should suffice too.